semantic-python/src/Language/Python/Core.hs

```diff
@@ -303,7 +297,7 @@ instance Compile Py.ParenthesizedExpression
 instance Compile Py.PassStatement where
   compileCC it@Py.PassStatement {} _ = locate it $ Core.unit
 
-deriving via CompileSum Py.PrimaryExpression instance Compile Py.PrimaryExpression
+deriving instance Compile Py.PrimaryExpression
```

Comment:
We can get these instances via `GeneralizedNewtypeDeriving` now thanks to @aymannadeem!